### PR TITLE
feat(EMS-1254-1260): Application submission - exporter document emails

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR adds the second and final email that gets sent to an exporter when an application is submitted.

The email template that is sent depends on certain answers to a couple of questions RE the buyer and declarations.

## Changes
- Add new email template IDs.
- Rename `sendEmailApplicationSubmitted` to `sendApplicationSubmittedEmails`.
- Add new `documentsEmail` function to the emails object.
- Rename/restructure `sendApplicationSubmittedEmails` into `applicationSubmittedEmails.send`, makes for simpler unit tests/stubbing.
- Add logic to `sendApplicationSubmittedEmails` so that it gets the application's declaration and depending on some answers, send the document email with the appropriate template.

## Other improvements
- Improved test coverage for `sendEmail` functions.
- Move `mockBuyer` into test mocks.
- Update mock declarations.
- Update `applicationSubmittedEmail` function to accept an object instead of string params - DRY.
- Update/align UI mock application.

### CODE REVIEW

File: src/api/emails/send-application-submitted-emails/index.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    if (!declaration) {
+      console.error('Sending application submitted emails to exporter - no declarations exist with the provided ID');
+
+      return {
+        success: false,
+      };
+    }
+
+    // generate email variables
+    const { email, firstName } = exporter;
```
Comment (abhi-markan) [inline src/api/emails/send-application-submitted-emails/index.ts:62]: I guess `email` and `firstName` will always exist?

**Review by abhi-markan — COMMENTED** [review]
Good work, only minor comments to address please.

File: src/api/.keystone/config.js [hunk 2]
```diff
@@ -1353,24 +1373,52 @@ var sendEmailApplicationSubmitted = async (context, accountId, buyerId, referenc
...
+      buyerName: buyer.companyOrOrganisationName
+    };
+    const submittedResponse = await emails_default.applicationSubmittedEmail(sendEmailVars);
+    if (!submittedResponse.success) {
+      throw new Error("Sending application submitted emails to exporter");
     }
-    throw new Error(`Sending email to exporter - application submitted ${emailResponse}`);
+    let documentsResponse;
+    let useAntiBriberyAndTradingHistoryTemplate = false;
+    if (buyer.exporterIsConnectedWithBuyer && buyer.exporterIsConnectedWithBuyer === "Yes") {
```
[the remark this answers is not in the document]
Comment (ttbarnes) [reply src/api/.keystone/config.js:1403]: They are loaded from constants but in this file (keystone compilation), it's compiled without the constant which makes sense

I.e where we actually define it, we have:

```js
const isConectedWithBuyer = buyer.exporterIsConnectedWithBuyer && buyer.exporterIsConnectedWithBuyer === ANSWERS.YES;
```

File: src/api/emails/send-application-submitted-emails/index.ts (hunk 1 above)
Comment (ttbarnes) [reply src/api/emails/send-application-submitted-emails/index.ts:62]: Yes, they will always exist - an account/exporter cannot be created without these, amongst other fields

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**